test/func_test/test_acnt.py

```diff
@@ -24,6 +28,53 @@ def supernode_addr(self) -> str:
         """
         return cft.SUPERNODE_ADDR
 
+    def test_pri_str_cons(self, chain: pv.Chain):
+        """
+        test_pri_str_cons tests method from_pri_key_str.
+
+        Args:
+            chain (pv.Chain): The chain.
+        """
+
+        acnt = pv.Account.from_pri_key_str(chain, self.pri_key)
+
+        assert acnt.key_pair.pri.data == self.pri_key
+        assert acnt.key_pair.pub.data == self.pub_key
+        assert acnt.addr.data == self.addr
+
```

Comment:
Can have another test case for testing constructing Account with private key model only.